tests/test_routes.py

```diff
@@ -556,17 +565,12 @@ def test_enable_disable_update_action_error_handler(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
         # Try disabling updates to product ID 3 (doesn't exist)
-        response = self.client.delete(
-            BASE_URL + "/3/NEW/active"
-        )
+        response = self.client.delete(BASE_URL + "/3/NEW/active")
 
-        # Regardless, HTTP_204_NO_CONTENT is returned
-        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
```

Comment:
I had to change this because of how Flask-RestX works. It was throwing a 404 NOT FOUND if you pass in something that doesn't exist

We're not really "deleting" anything in this endpoint, so I'm not sure if it was even appropriate to return a 204 NO CONTENT to begin with?